www/include/Administration/configChangelog/viewLogs.ihtml

```diff
@@ -90,3 +90,9 @@
 <input type='hidden' id='limit' name='limit' value='{$limit}'>
 {$form.hidden}
 </form>
+
+{literal}
+<script>
+    formatDateMoment();
+</script>
+{/literal}
```

Comment:
It is not possible to call the function directly after the bracket? This will avoid closing the "script" and "literal" tags to open them again afterwards.